src/it/resources/com/google/checkstyle/test/chapter5naming/rule51identifiernames/InputCatchParameterName.java

```diff
@@ -1,65 +1,99 @@
-        } catch (InterruptedException ie) {
-        }
-        try {
-        } catch (Exception ok) {
-            // appropriate to take no action here
-        }
-        try {
-        } catch (Exception e1) {
-            try {
-            } catch (Exception e2) {
-            }
-        }
-        try {
-        } catch (Throwable t1) {
-            try {
-            } catch (Throwable t2) {
-            }
-        }
-        try {
-        } catch (Exception iException) { // warn
-        }
-        try {
-        } catch (Exception ex_1) { // warn
-        }
-        try {
-        } catch (Exception eX) { // warn
-        }
-        try {
-        } catch (Exception eXX) { // warn
-        }
-        try {
-        } catch (Exception x_y_z) { // warn
-        }
-        try {
-        } catch (Exception Ex) { // warn
-        }
+  {
+    try {
+      /* foo */
+    } catch (Exception e) {
+      /* foo */
     }
+    try {
+      /* foo */
+    } catch (Exception ex) {
+      /* foo */
+    }
+    try {
+      /* foo */
+    } catch (Error | Exception err) {
+      /* foo */
+    }
+    try {
+      /* foo */
+    } catch (Exception exception) {
+      /* foo */
+    }
+    try {
+      /* foo */
+    } catch (Exception exception1) {
+      /* foo */
+    }
+    try {
+      /* foo */
+    } catch (Exception noWorries) {
+      /* foo */
+    }
+    try {
+      /* foo */
+    } catch (Throwable t) {
+      /* foo */
+    }
+    try {
+      throw new InterruptedException("interruptedException");
+    } catch (InterruptedException ie) {
+      /* foo */
+    }
+    try {
+      /* foo */
+    } catch (Exception ok) {
+      // appropriate to take no action here
+    }
+    try {
+      /* foo */
+    } catch (Exception e1) {
+      try {
+        /* foo */
+      } catch (Exception e2) {
+        /* foo */
+      }
+    }
+    try {
+      /* foo */
+    } catch (Throwable t1) {
+      try {
+        /* foo */
+      } catch (Throwable t2) {
+        /* foo */
+      }
+    }
+    try {
+      /* foo */
+    } catch (Exception iException) {
+      // violation above 'Catch parameter name 'iException' must match pattern'
+    }
+    try {
+      /* foo */
+    } catch (Exception ex_1) { // violation 'Catch parameter name 'ex_1' must match pattern'
+    }
+    try {
+      /* foo */
+    } catch (Exception eX) { // violation 'Catch parameter name 'eX' must match pattern'
+    }
+    try {
+      /* foo */
+    } catch (Exception eXX) {
+      // 2 violations above:
+      //  'Abbreviation in name 'eXX' must contain no more than '1' consecutive capital letters.'
+      //  'Catch parameter name 'eXX' must match pattern'
+      /* foo */
+    }
+    try {
+      /* foo */
+    } catch (Exception x_y_z) { // violation 'Catch parameter name 'x_y_z' must match pattern'
+    }
+    try {
+      /* foo */
+    } catch (Exception Ex) { // violation 'Catch parameter name 'Ex' must match pattern'
```

Comment:
Please do us a favor and add what is written in style guide `name_, mName, s_name and kName.`
https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names

It might be better to make new input, as existing is too much focused on Exceptions.

Reply:
done 

https://github.com/checkstyle/checkstyle/blob/603d83e65807027e570f63f39d64c087ffe30b41/src/it/resources/com/google/checkstyle/test/chapter5naming/rule51identifiernames/InputCatchParameterName.java#L98-L113